### ISSUE TITLE
Skip automated SVA analysis

### ISSUE THREAD

**Issue by tbrittoborges** [kvittingseerup/IsoformSwitchAnalyzeR#178]
Hi @kvittingseerup ,
Correct if I am wrong, there is no way to bypass `Step 5 of 10: Testing for unwanted effects...`, the SVA analysis? The only way to do this is installing the older version `1.17.05`.

**Comment by kvittingseerup** [comment]
You are right, there is currently no way of skipping the sva analysis. But why would you want to skip that step?

I cannot recommend using an older version. As you can see from the NEWS file a lot of updates and bugs have been fixed since then.

**Comment by tbrittoborges** [comment]
Hi Kristoffer,
Thank you for the prompt response. You are right, using an older version won't work due to the changes in the "dplyr` join API. 

> But why would you want to skip that step?

In my experience, using [num.sv](https://rdrr.io/bioc/sva/man/num.sv.html) is not usually great as it can capture some of the biological signal, depending on the experiment design and experiment. In general, selecting the number of variables is [hard](https://support.bioconductor.org/p/71447/#87007), unless you have an idea of the metadata for batches, sequencing lanes, RNA integrity, etc., so one can estimate their impact on the SVs - I know, in that case you don't really need SV, but they can help if the metadata is incomplete! 

Although I think SVA is a good addition to ISAR, I think it should be an optional step prior to the data load step, and it should come with methods to help for selecting the numb. of SVs (PCA, correlations of SV and metadata, etc.). So please consider making the SVA step an optional step. I have a patch, and we discuss a PR.
Thanks again!

**Comment by tbrittoborges** [comment]
See also [here](https://biodatascience.github.io/compbio/dist/sva.html):

>This procedure can be used to estimate unknown, or undocumented batches present in high-throughput datasets, although we must be careful not to overfit and remove too much of the variation in the data – some of which represents natural biological variations of samples.
